src/material/slider/slider-input.ts

```diff
@@ -439,12 +439,11 @@ export class MatSliderThumb implements _MatSliderThumb, OnDestroy, ControlValueA
     }
   }
 
-  _onPointerUp(event: PointerEvent): void {
+  _onPointerUp(): void {
     this._isActive = false;
-    this._updateWidthInactive();
-    if (!this.disabled) {
-      this._handleValueCorrection(event);
-    }
+    setTimeout(() => {
+      this._updateWidthInactive();
+    }, 10);
```

Comment:
what's the deal with this `10`?

Reply:
If there's no `setTimeout` here, the `pointerup` value gets recalculated based on the inactive width of the hidden slider input. I double checked and found that I could just not provide the time arg to the setTimeout here, so I sent in a commit with that. Let me know if you're confused about the active vs inactive width because I'm not sure if we've ever discussed it in depth and it's kinda a confusing topic on it's own